### PR TITLE
Resize the composer based on the newlines in the text

### PR DESCRIPTION
**Pull request by Jikstra**
screenshot_20190131_162940

Still missing:
- [x] emoji picker isn't at the right position currently
- [x] resizing isn't on point yet
- [x] we sometimes have scrollbars even if we don't need them
- [x] composer buttons should stick to the bottom (position them absolutely)
- [x] some weird css bugs if we're havent selected a chat yet
- [x] resizing composer is getting called too often

Fixes https://github.com/deltachat/deltachat-desktop/issues/543

### CODE REVIEW

File: src/renderer/components/ChatView.js [hunk 1]
```diff
@@ -87,7 +91,7 @@ class ChatView extends React.Component {
     this.lastId = this.props.chat.id
     this.previousScrollHeightMinusTop = null
 
-    this.composerRef = React.createRef()
+    this.conversationRef = React.createRef()
```
Comment (ralphtheninja) [inline src/renderer/components/ChatView.js]: This breaks setting focus on the composer, used in `SplittedChatListAndView.js`.

Comment (ralphtheninja) [reply src/renderer/components/ChatView.js]: cc @Simon-Laux

Comment (Simon-Laux) [reply src/renderer/components/ChatView.js]: If that works, do it.

**Comment by Jikstra** [comment]
Just found a bug with the three dots menu, the top part is hidden. I'm trying to fix it.

**Pull request merged**